doc/cffi-manual.texinfo

```diff
@@ -2855,6 +2859,14 @@ being passed as arguments to or returned from foreign functions,
 respectively. The same applies to any other situations where an object
 of an @code{enum} type is expected.
 
+If a value should be translated to lisp that is not declared in the
+enum, an error will be signalled. You can elide this error and instead
+make it pass the original enum value by specifying
+@var{undeclared-values-allowed-p}. This can be useful for very large
```

Comment:
minor style suggestion: maybe rename this to `allow-undeclared-values` à la `allow-other-keys`?

Reply:
I wasn't sure whether the 'predicates should be -p things' conventions should be followed or not. I'll change it to what you suggested.

Reply:
That convention usually applies to function names, not argument names, I think.